include/room_member.h

```diff
@@ -0,0 +1,95 @@
+// Copyright 2017 Tunnler Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+//
+
+#ifndef room_member_h
+#define room_member_h
+
+#include <deque>
+#include <string>
+
+#include "tunnler.h"
+
+#include "RakNetStatistics.h"
+#include "RakPeerInterface.h"
+
+class RoomMember {
```

Comment:
Throughout the files, please use spaces instead of tabs for indentation, 1 tab = 4 spaces